app/GameMissions/Abstracts/GameMission.php

```diff
@@ -274,6 +275,8 @@ public function start(PlanetService $planet, Coordinate $targetCoordinate, Plane
         if ($mission->time_arrival < Carbon::now()->timestamp) {
             $this->process($mission);
         }
+        $arrival_time_to_carbon = \Carbon\Carbon::parse($mission->time_arrival);
```

Comment:
```suggestion
        $arrival_time_to_carbon = Carbon::parse($mission->time_arrival);
```
This should be fine, as we already import it.